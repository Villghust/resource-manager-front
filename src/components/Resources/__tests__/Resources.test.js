import React from 'react';
import renderer from 'react-test-renderer';

import { ListResources } from '../index';

it('renders correctly', () => {
    const tree = renderer.create(<ListResources />).toJSON();
    expect(tree).toMatchSnapshot();
});
