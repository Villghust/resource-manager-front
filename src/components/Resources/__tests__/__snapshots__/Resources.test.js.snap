// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<span
  className="MuiSkeleton-root MuiSkeleton-rect MuiSkeleton-pulse"
  style={
    Object {
      "height": 250,
      "width": undefined,
    }
  }
/>
`;
